keyboards/flx/lodestone/info.json

```diff
@@ -0,0 +1,84 @@
+"keyboard_name": "lodestone", 
+    "url": "https://prototypist.net/", 
+    "maintainer": "Flex", 
+    "width": 16, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
+	        "layout": [
+	            {"label":"K00 (B3,B2)", "x":0, "y":0},
+	            {"label":"K01 (B3,F5)", "x":1, "y":0},
+	            {"label":"K02 (B3,F6)", "x":2, "y":0},
+	            {"label":"K03 (B3,D0)", "x":3, "y":0},
+	            {"label":"K04 (B3,D1)", "x":4, "y":0},
+	            {"label":"K05 (B3,D2)", "x":5, "y":0},
+	            {"label":"K06 (B3,D3)", "x":6, "y":0},
+	            {"label":"K07 (B3,D5)", "x":7, "y":0},
+	            {"label":"K08 (B3,D4)", "x":8, "y":0},
+	            {"label":"K09 (B3,D6)", "x":9, "y":0},
+	            {"label":"K0A (B3,D7)", "x":10, "y":0},
+	            {"label":"K0B (B3,B4)", "x":11, "y":0},
+	            {"label":"K0C (B3,B5)", "x":12, "y":0},
+	            {"label":"K0D (B3,B6)", "x":13, "y":0},
+	            {"label":"K0E (B3,C6)", "x":14, "y":0},
+	            {"label":"K0F (B3,C7)", "x":15, "y":0},
+	            {"label":"K10 (B7,B2)", "x":0, "y":1, "w":1.5},
+	            {"label":"K11 (B7,F5)", "x":1.5, "y":1},
+	            {"label":"K12 (B7,F6)", "x":2.5, "y":1},
+	            {"label":"K13 (B7,D0)", "x":3.5, "y":1},
+	            {"label":"K14 (B7,D1)", "x":4.5, "y":1},
+	            {"label":"K15 (B7,D2)", "x":5.5, "y":1},
+	            {"label":"K16 (B7,D3)", "x":6.5, "y":1},
+	            {"label":"K17 (B7,D5)", "x":7.5, "y":1},
+	            {"label":"K18 (B7,D4)", "x":8.5, "y":1},
+	            {"label":"K19 (B7,D6)", "x":9.5, "y":1},
+	            {"label":"K1A (B7,D7)", "x":10.5, "y":1},
+	            {"label":"K1B (B7,B4)", "x":11.5, "y":1},
+	            {"label":"K1C (B7,B5)", "x":12.5, "y":1},
+	            {"label":"K1D (B7,B6)", "x":13.5, "y":1, "w":1.5},
+	            {"label":"K1F (B7,C7)", "x":15, "y":1},
+	            {"label":"K20 (F0,B2)", "x":0, "y":2, "w":1.75},
+	            {"label":"K21 (F0,F5)", "x":1.75, "y":2},
+	            {"label":"K22 (F0,F6)", "x":2.75, "y":2},
+	            {"label":"K23 (F0,D0)", "x":3.75, "y":2},
+	            {"label":"K24 (F0,D1)", "x":4.75, "y":2},
+	            {"label":"K25 (F0,D2)", "x":5.75, "y":2},
+	            {"label":"K26 (F0,D3)", "x":6.75, "y":2},
+	            {"label":"K27 (F0,D5)", "x":7.75, "y":2},
+	            {"label":"K28 (F0,D4)", "x":8.75, "y":2},
+	            {"label":"K29 (F0,D6)", "x":9.75, "y":2},
+	            {"label":"K2A (F0,D7)", "x":10.75, "y":2},
+	            {"label":"K2B (F0,B4)", "x":11.75, "y":2},
+	            {"label":"K2C (F0,B5)", "x":12.75, "y":2},
+	            {"label":"K2D (F0,B6)", "x":12.75, "y":2, "w":2.25},
```

Comment:
```suggestion
	            {"label":"K2D (F0,B6)", "x":13.75, "y":2, "w":1.25},
```

Reply:
Is this a case where it may be better to have a separate ISO layout? Just for clarity in the code?
K2C = Iso #
K2D = ANSI Enter

I'll commit this, but let me know which would be preferred practice?

Reply:
> Is this a case where it may be better to have a separate ISO layout? Just for clarity in the code?
> K2C = Iso #
> K2D = ANSI Enter

I can add this to the codebase really easily from that information actually. If you're only going to include one layout macro use `LAYOUT` if the PCB has a fixed layout, or `LAYOUT_all` if there are layout options (user can choose between ANSI and ISO, or split/2u Backspace; things like that).